### PR TITLE
created API connection with frontend

### PR DESCRIPTION
**Pull request by dmkainos**

### CODE REVIEW

File: validator/orderValidator.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+const dallan2:number=0;
```
Comment (glenn-hamilton) [inline validator/orderValidator.ts]: Is this needed?

File: public/script.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+//code for submission
```
Comment (glenn-hamilton) [inline public/script.js]: Is this needed?

File: app.ts [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+   }
+}
+
+app.listen(3000, ()=>{
+    console.log('Server is running on port 3000')
+});
+
+//Express bottles
+app.get('/', (req:Request,res:Response)=>{
+    res.render('pizza',{
```
Comment (glenn-hamilton) [inline app.ts:45]: Is there a way of testing that this works without using `Pizza Time`?

**Comment by ghost** [comment]
Made changes

File: views/pizza.html
Comment (shaunganley) [inline views/pizza.html:1]: do you need this?

File: app.ts [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+        deliveryEmployee: DeliveryEmployee;
+   }
+}
+
+app.listen(3000, ()=>{
+    console.log('Server is running on port 3000')
+});
+
+//Express bottles
+app.get('/', (req:Request,res:Response)=>{
```
Comment (shaunganley) [inline app.ts:44]: do you need this?

**Review by shaunganley — APPROVED** [review]

**Pull request merged**